jax/_src/scipy/linalg.py

```diff
@@ -103,7 +103,23 @@ def eigh(a, b=None, lower=True, eigvals_only=False, overwrite_a=False,
   del overwrite_a, overwrite_b, turbo, check_finite
   return _eigh(a, b, lower, eigvals_only, eigvals, type)
 
+@partial(jit, static_argnames=('output',))
+def _schur(a, output):
+  if output != "real":
```

Comment:
This should raise an error if an invalid `output` is provided

Reply:
I added a line to check the `output` argument `schur`.